CHANGELOG.txt

```diff
@@ -398,7 +398,7 @@ Misc/Internal
 - Move Dockerfile to Dockerfile.ci (#7829)
 - Generate requirements are now sorted (#8040)
 - Change name of the common environment initialization function (#7805)
-- Requirements now depend on python version (#7841)
+- Requirements now depend on Python version (#7841)
```

Comment:
Those refer to commit messages that cannot be changed so we better do not change them.